PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/webapp/WebAppPropertyView.java

```diff
@@ -20,13 +40,9 @@ public static WebAppBasePropertyView create(@NotNull final Project project, @Not
 
     private WebAppPropertyView(@NotNull final Project project, @NotNull final String sid,
                                @NotNull final String webAppId) {
-        super(project, sid, webAppId);
+        super(project, sid, webAppId, null);
+        this.id = "com.microsoft.intellij.helpers.webapp.WebAppBasePropertyView";
```

Comment:
the id should be set via an interface.

Reply:
Will make the base class an abstract class and define the abstract methods for children class to overwrite.